R/wb_load.R

```diff
@@ -1674,6 +1686,25 @@ wb_load <- function(
     wb$workbook$xti <- NULL
   }
 
+  do_not_remove <- c(
+    if (length(wb$blob)) paste0(basename2(wb$blob)) else NULL,
```

Comment:
Should be: `paste0(basename2(wb$blob), collapse = "|")`, because ws$blob could contain multiple blobs